tools/ios/build_xcframework.py

```diff
@@ -213,19 +212,61 @@ def platform_arch_framework_build_dir(platform, arch):
             platform_fat_framework_dir.mkdir()
 
             # copy over files from arch-specific framework to fat framework
-            for framework_file_relative_path in [Path("Headers"), Path("Info.plist")]:
-                src = first_arch_framework_dir / framework_file_relative_path
-                dst = platform_fat_framework_dir / framework_file_relative_path
-                if src.is_dir():
-                    shutil.copytree(src, dst)
-                else:
-                    shutil.copy(src, dst)
-
-            # combine arch-specific framework libraries
-            arch_libs = [str(framework_dir / "onnxruntime_extensions") for framework_dir in arch_framework_dirs]
-            run(
-                *([_lipo, "-create", "-output", str(platform_fat_framework_dir / "onnxruntime_extensions")] + arch_libs)
-            )
+            # macos requires different framework structure:
+            # https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPFrameworks/Concepts/FrameworkAnatomy.html
+            if platform == "macosx" or platform == "maccatalyst":
```

Comment:
we used `macabi` in the ORT version. just checking that `maccatalyst` is correct here.

Reply:
build_xcframework.py is slightly inconsistent with onnxruntime build xcframework script. It defines: 
```
_all_supported_platform_archs = {
    "iphoneos": ["arm64"],
    "iphonesimulator": ["x86_64", "arm64"],
    "macosx": ["x86_64", "arm64"],
    "maccatalyst": ["x86_64", "arm64"],
}
```